packages/cli/src/utils/user.ts

```diff
@@ -23,7 +22,9 @@ export const getUser = async (instanceUrl: string): Promise<UserInfo | undefined
         );
         return user.data.currentUser;
     } catch {
-        Logger.error(`You are not logged in, you can use the command ${pc.bold('frontify-cli login')}.`);
+        Logger.error(
+            `You are currently not logged in. You can use the command ${pc.bold('frontify-cli login')} to log in.`,
```

Comment:
Maybe here the same error as before?

Reply:
That was my initial thought as well. But `getUser` gets called by the `login` command which would cause confusion since the `login` command can't deal with the token being passed directly. In connection with the `deploy` command, the only case where this would happen is when something's wrong with the stored credentials, so I propose we keep it like that?

Reply:
Sounds good